docs/src/modules/components/Link.tsx

```diff
@@ -93,7 +93,8 @@ const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(function Link(props,
     userLanguage !== 'en' &&
     pathname &&
     pathname.indexOf('/') === 0 &&
-    pathname.indexOf('/blog') !== 0
+    pathname.indexOf('/blog') !== 0 &&
+    !pathname.startsWith(`/${userLanguage}/`)
```

Comment:
How is this related to the changes?

Reply:
The change in `href` (Link consider it as pathname) can contain `userLanguage` because it is in the URL.

if we don't check the `userLanguage` here, when I click on the NavItem, I will see this on the URL
```
/zh/zh/getting-started/installation/#npm
```